low-level/solution/lr2/server/health.py

```diff
@@ -129,6 +130,7 @@ def get_data(self, rpath):
             logger.error(self.log.svc_log(f"{msg}"))
             raise ResourceMapError(errno.EINVAL, msg)
 
+        info = Utility.normalize_kv(info, "NA", "Not Available")
```

Comment:
Change the schema itself to reflect "Not Available" where "NA" is set.

Reply:
Sometimes we will get "N/A" as a value from source we have to apply more filtration logic there.
Instead of we can make that all change in one place.
That's I have introduced a new generic function in the utils class. which will take care of these things.